<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Get_Patient_Inf_Dialog</class>
 <widget class="QDialog" name="Get_Patient_Inf_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病人信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="bed_nub_l">
       <property name="text">
        <string>病床号：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="age_text"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="gender_text"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_text"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="take_pack_I">
       <property name="text">
        <string>采取部位:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="take_pack_text"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="name_l">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="bed_nub_text"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox_l">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gender_l">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="age_l">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="picture_number_I">
       <property name="text">
        <string>涂片号:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="picture_number_text"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_l</sender>
   <signal>accepted()</signal>
   <receiver>Get_Patient_Inf_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_l</sender>
   <signal>rejected()</signal>
   <receiver>Get_Patient_Inf_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
